File: benchmark/python/custom_randy.py
import random

mylist = []

for i in range(10000):
    mylist.append(random.randint(0, 100))


def bubble_sort(items):
    n = len(items) - 1
    for i in range(n):
        for j in range(n-i):
            if items[j] > items[j+1]:
                items[j], items[j+1] = items[j+1], items[j]
    return items

bubble_sort(mylist)

File: benchmark/golang/custom_randy.go
package main

import (
    "math/rand"
)


func bubble_sort(items []int) []int {
    n := len(items) - 1
    for i := 0; i < n; i++ {
	for j := 0; i < n-i; j++ {
	    if items[j] > items[j+1] {
		items[j], items[j+1] = items[j+1], items[j]
	    }
	}
    }
    return items
}

func main() {
    var mylist []int

    for i := 0; i < 10000; i++ {
	mylist = append(mylist, rand.Intn(100))
    }

    bubble_sort(mylist)
}
